### ISSUE TITLE
[🐞] Getting "Failed to construct 'URL': Invalid URL" on nx monorepo on dev serve

### ISSUE THREAD

**Issue by agCkre** [QwikDev/qwik#6617]
### Which component is affected?

Qwik Runtime

### Describe the bug

In dev mode, running `nx serve` command, when hovering the button I get the following error

`Failed to construct 'URL': Invalid URL. `

![Screenshot 2024-06-29 at 01 04 01](https://github.com/QwikDev/qwik/assets/126238538/feac96e2-6a7a-4ba5-bde9-e333b2ea1986)

### Reproduction

https://github.com/chrisKrezias/nx-qwik-test-repo

### Steps to reproduce

Run `npm install` and `npx nx serve todo`

### System Info

```shell
System:
    OS: macOS 14.4.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 49.16 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.11.1 - ~/.volta/tools/image/node/20.11.1/bin/node
    Yarn: 4.0.0-rc.50 - ~/.volta/tools/image/yarn/4.1.1/bin/yarn
    npm: 10.2.4 - ~/.volta/tools/image/node/20.11.1/bin/npm
    pnpm: 8.15.3 - ~/.volta/bin/pnpm
  Browsers:
    Chrome: 126.0.6478.127
    Safari: 17.4.1
  npmPackages:
    @builder.io/qwik: 1.6.0 => 1.6.0 
    @builder.io/qwik-city: 1.6.0 => 1.6.0 
    nx: 19.3.2 => 19.3.2 
    qwik-nx: ^2.3.0 => 2.3.0 
    typescript: ~5.4.2 => 5.4.5 
    undici: ^5.22.0 => 5.28.4 
    vite: ~5.2.0 => 5.2.13
```

**Comment by gioboa** [comment]
Hi @agCkre if you click on index and add a breakpoint 
![Screenshot_20240629-033613](https://github.com/QwikDev/qwik/assets/35845425/6646c351-20f1-414f-93fb-04a9aa2fa434)
Which URL is trying to create?

**Comment by agCkre** [comment]
This is the qrl `"//%40fs//project/workspace/node_modules/%40builder.io/qwik-city/link_component_handleprefetch_osdg8fnytw4.js?_qrl_parent=/project/workspace/node_modules/%40builder.io/qwik-city/index.qwik.mjs#Link_component_handlePrefetch_Osdg8FnYTw4"`

If it helps I have deployed here https://9qg4kd-5173.csb.app/

Whenever you hover on the button the error should appear


https://github.com/QwikDev/qwik/assets/126238538/abf3644e-cf4a-46e1-b3f6-78b01ad14dd4

**Comment by yuran1811** [comment]
I have the same issue when running my qwik project with yarn dev, but it stills be ok when yarn preview

**Comment by gioboa** [comment]
Can you try with [this build](https://github.com/QwikDev/qwik/runs/26865227218) pls?

**Comment by yuran1811** [comment]
> Can you try with [this build](https://github.com/QwikDev/qwik/runs/26865227218) pls?

I try this and get this error 
![image](https://github.com/QwikDev/qwik/assets/76398420/dfc1db43-0e5d-4d0f-94f8-f990d6a5ae1a)

**Comment by gustavocadev** [comment]
Same here in Qwik 1.6.0 with one project I'm building

**Comment by wmertens** [comment]
Try with the package from https://github.com/QwikDev/qwik/pull/6629#issuecomment-2201091318

**Comment by Loshido** [comment]
switching to 1.5.7 seems to work

-> `npm uninstall @builder.io/qwik @builder.io/qwik-city eslint-plugin-qwik`
-> `npm i @builder.io/qwik@1.5.7 @builder.io/qwik-city@1.5.7 eslint-plugin-qwik@1.5.7 -D`

**Comment by wmertens** [comment]
@Loshido please try https://github.com/QwikDev/qwik/issues/6617#issuecomment-2201096235

**Comment by nelsonprsousa** [comment]
I noticed this error once I upgrade to v1.6.0 as well.

I just tested #6629, but got some problems related with some other dependency (in this case, `qwik-speak`).

@wmertens does this PR adds some breaking change?

**Comment by wmertens** [comment]
@nelsonprsousa no internationally breaking changes. Did you try reloading a few times?

It would help if you set `debug:true` in the qwikVite plugin and then find where it's trying to resolve that file. Basically anywhere that mentions that hash 6bh9...

Please copy/paste, don't screenshot, and do it in #6629

**Comment by shairez** [comment]
This should be fixed by #6629 

Thanks @wmertens ! 🙏
